docs/guides/recovery/user-recovery.primary.in.rst

```diff
@@ -0,0 +1,67 @@
+Troubleshooting advice for RSP login issues.
+
+
+Problems with a new account
+===========================
+
+Attempts to login will not be successful until your account petition is approved by Rubin staff
+(i.e., step 8 of the process for :doc:`../getting-started/get-an-account`).
+This can take up to a week.
+Your patience is appreciated.
+
+Sometimes this approval process requires that Rubin staff verify that you have Rubin data rights.
+Please check your email's spam folder for a data rights verification
+request from `Heather Shaughnessy <mailto:sheather@slac.stanford.edu>`__.
```

Comment:
Let's not bake in Heather's name and email address in this. We can set up an alias like account-help@lists.lsst.org that goes to Heather's email.